### DISCUSSION TITLE
cant make my c# script work

### DISCUSSION THREAD

**Discussion by N1emiec69** [ORelio/Minecraft-Console-Client#2633] [Q&A]
hi i made a script which use item in left hand every 5 seconds
i dont get any errors it just dont work

//MCCScript 1.0

MCC.LoadBot(new YourBot());

//MCCScript Extensions

class YourBot : ChatBot
{
    public void UseItemInLeftHand()
    {
      UseItemInLeftHand(); 
        Thread.Sleep(5000);
        }
 }

**Comment by yaggod** [comment]
Obviously that doesn't work. What you want is to look at ChatBot api (there is not actual docs for it, but you can watch for virtual methods in ChatBot class [here](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Scripting/ChatBot.cs)) and find any methods called once per while, i.e. Update() method, override it in your class (in YourBot class) and then call method you want to once per 50 Update calls
